ksh93/PKGBUILD

```
@@ -0,0 +1,124 @@
+# This took a lot of trial and error. There possibly should be patches from the linked gisburn repo and the ones from debian and red hat.
+#
+pkgname=ksh93
+pkgver=1.0.10
+pkgrel=1
+pkgdesc="The Original ATT Korn Shell, Now Maintained By The Community."
+arch=('x86_64')
+url="https://www.github.com/ksh93/ksh"
+license=('spdx:EPL-2.0')
+
+# These makedeps were suggested by 'gisburn' in a ksh93 issue.
+makedepends=(
+	'gcc'
+	'gdb'
+	'make'
+	'gettext'
+	'gettext-devel'
+	'flex'
+	'bison'
+	'unzip'
+	'pax'
+	'tar'
+	'libiconv-devel'
+	'ncurses-devel'
+	'gmp-devel'
+	'mpfr-devel'
+	'mpc-devel'
+	'isl-devel'
+	'procps-ng'
+)
```

There's no way all of these are actually needed.

---

I don't know what I was doing honestly. I just saw someone else from the ksh93 repo say you should install that and just dropped it in.

---

```suggestion
makedepends=(
	'gcc'
	'gettext-devel'
	'libiconv-devel'
	'ncurses-devel'
)
```

maybe adding back
```
	'flex'
	'bison'
	'gmp-devel'
	'mpfr-devel'
	'mpc-devel'
	'isl-devel'
```
if it's demonstrated they're actually used

---

I'm not going to include libiconv-devel because it completely fails if the GNU one is installed, I have to patch out looking for the system wide libiconv and use the bundled one instead.